### ISSUE TITLE
Raising a DoesNotExist class gives E0702: Raising NoneType while only classes or instances are allowed (raising-bad-type)

### ISSUE THREAD

**Issue by mikebryant** [landscapeio/pylint-django#46]
If you have a model ```SomeModel``` and in your code you use ```raise SomeModel.DoesNotExist```, due to the transformation in ```pylint_django/transforms/transforms/django_db_models.py``` pylint raises E0702.

This code should raise no errors.
